vesc_hw_interface/src/vesc_servo_controller.cpp

```diff
@@ -288,33 +322,35 @@ void VescServoController::updateSensor(const std::shared_ptr<VescPacket const>&
     std::shared_ptr<VescPacketValues const> values = std::dynamic_pointer_cast<VescPacketValues const>(packet);
     const double current = values->getMotorCurrent();
     const double velocity_rpm = values->getVelocityERPM() / static_cast<double>(num_rotor_poles_ / 2);
-    const int steps = static_cast<int>(values->getPosition());
-    if (initialize_)
+    const int16_t steps = static_cast<int16_t>(values->getPosition());
```

Comment:
The part was uint16_t before editing, so I changed it to int16_t.
It may also work with int32_t, but I have not verified this.
If it works with int32_t we will need to raise an issue and make the change.

Reply:
I don't know if we can rely on this but vesc_packet seems to return int32_t 
https://github.com/sbgisen/vesc/blob/3d5e62e736d2c3f92d26e006b3ccf48f5f5c4dc4/vesc_driver/src/vesc_packet.cpp#L309-L324

Reply:
I checked operation near zero steps, but not until int32_t was circled. It is good as far as I can see, so I will include it in this PR.